_data/colors.yml

```diff
@@ -0,0 +1,74 @@
+
+# Color configurations
+colors:
```

Comment:
This top level `colors:` can be removed and then it can be referenced as `site.data.colors.colors` vs `site.data.colors`.